README.md

```diff
@@ -118,6 +119,21 @@ Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 We conduct weekly bug triage calls at our slack channel on Tuesdays.
 More details are available [here](https://github.com/ceph/ceph-csi/issues/463)
 
+## Community Meeting
+
+A regular community meeting takes place every other [Monday,Tuesday,Thursday at
+7:30 PM IST (Indian Standard Time)](https://redhat.bluejeans.com/702977652).
```

Comment:
Please put the time in UTC and add a link to some time conversion website, in addition to the `date -d '14:00 UTC'` command to convert it to local time.

Reply:
@Madhu-1 as disscssed please add https://docs.google.com/document/d/1wgRPWRt6auP873MAwEBuEqwtlg3O9eGqrFH9YuADxHc/edit# details in the doc .. the daily standup meetings or team meetings which we have daily can be renamed to `cst team` or `dev` meeting.

Reply:
@humblec do you want to use the above google doc link in this PR instead of https://github.com/ceph/ceph-csi/pull/994/files#diff-04c6e90faac2675aa89e2176d2eec7d8R135

Reply:
@nixpanic i will move the time to UTC, won't mention anything about the time conversion website, let the user do the conversion. sounds good?

Reply:
The idea is to make it easier by linking to one of the conversion sites with pre-filled time in UTC. But if you really do not like it, at least included the `date` commandline

Reply:
> @humblec do you want to use the above google doc link in this PR instead of https://github.com/ceph/ceph-csi/pull/994/files#diff-04c6e90faac2675aa89e2176d2eec7d8R135

No.. let `Dev team stand up` and also `community meeting` meeting doc remain seperate , but what I was requesting  here is, mention both of these meeting in this PR

Reply:
The meeting on Wednesday is Red Hat internal and for product discussions, it should not be mentioned as a 'community meeting' as not everyone from the community can join (or access the document).

Reply:
i updated the PR for UTC time. @humblec i have updated the doc for dev standup, as i don't know the timings for the community standup can you  please send a PR with more details on the community meeting

Reply:
@Madhu-1 the meeting timing is mentioned in the same doc :
It is `Meetings are held on Zoom every other Thursday at 9 AM PT (12 PM ET) to 10 AM PT (1 PM ET).  `

Reply:
I don't know this time works for all (at least for me 9:30 PM IST is too late), let's work on the timings first, if everyone agrees we can make a separate PR for the same

Reply:
Sure.. let me send it as a seperate PR.. Approving this.